_lifeofinternatpuppet/post_16.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: Week 16
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 16
+---
+
+Hello Everyone!
+
+Not too long to Christmas now, of course?
+
+This has been the busiest week of my placement so far, before the team wraps up for Christmas holidays!
+
+Some of the key highlights included:-
+
+**Module Release Prep**
+- [puppetlabs-motd (IAC-1308)](https://tickets.puppetlabs.com/browse/IAC-1308) (v4.3.0)
+- [puppetlabs-tomcat (IAC-1330)](https://tickets.puppetlabs.com/browse/IAC-1330)
+- [puppetlabs-tomcat (PR #426)](https://github.com/puppetlabs/puppetlabs-tomcat/pull/426) (v4.3.0)
+- [puppetlabs-tagmail (IAC-1349)](https://tickets.puppetlabs.com/browse/IAC-1349)
+- [puppetlabs-tagmail (PR #196)](https://github.com/puppetlabs/puppetlabs-tagmail/pull/196)
+
+**Module Releases**
+-[puppetlabs/accounts](https://forge.puppet.com/puppetlabs/accounts) (v6.4.0)
```

Comment:
puppetlabs/accounts isnt consistent with the other module names with '-'

Reply:
That's how @sheenaajay told me to format it

Reply:
@Disha-maker we can make it consistent with other releases you mentioned below.